client/src/components/EventMatrix.tsx

```diff
@@ -430,6 +430,7 @@ const EventMatrix = ({
                         leaf={task}
                         ascendantObjects={task.ascendantTasks}
                         parentField="parentTask"
+                        hideParents={taskUuid !== task.uuid}
```

Comment:
This hides the breadcrumb trail from the top-level objectives event matrix. Suggest to change to:
```javascript
                        hideParents={includeTask && taskUuid !== task.uuid}
```